lib/order.rb

```diff
@@ -0,0 +1,27 @@
+require 'menu'
+
+class Order
+    attr_reader :order
+
+    def initialize
+        @menu = Menu.new
```

Comment:
could have (@menu = menu || Menu.new) that way you could of used doubles for tests

Reply:
Please change `initialize`, so that you can inject a menu into an order. For example:
```ruby
menu = Menu.new
order = Order.new(menu)
```
Then you will be able to use test doubles instead of real `Menu` instances in your Order tests.

Reply:
Changed the order tests from subject to order double